TensorMath.lua

```diff
@@ -257,6 +211,7 @@ for _,Tensor in ipairs({"ByteTensor", "CharTensor",
              end
    end
 
+   if Tensor ~= 'HalfTensor' then
```

Comment:
@gchanan : adding  torch.hashalfmath literally does not compile: how do I refer to it correctly from this module ?